201-logic-app-xslt-with-params/artifacts/sample-input.xml

```diff
@@ -0,0 +1 @@
+﻿<ns0:Root xmlns:ns0="http://Integration.Source" />
```

Comment:
These are sample input payloads that I generally put under artifacts folder. Users can use this input data to fire the logic app.